src/Content/MailArchive/MailArchiveAttachmentCollection.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Frosh\MailArchive\Content\MailArchive;
+
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+
+/**
+ * @extends EntityCollection<MailArchiveAttachmentEntity>
+ */
+class MailArchiveAttachmentCollection extends EntityCollection
```

Comment:
what about we don't create collections and use the already existing EntityCollection?

Reply:
Uhhhh, this sounds really nice!